xds/src/test/java/io/grpc/xds/FilterChainMatchingProtocolNegotiatorsTest.java

```diff
@@ -88,6 +91,8 @@
   private static final String LOCAL_IP = "10.1.2.3";  // dest
   private static final String REMOTE_IP = "10.4.2.3"; // source
   private static final int PORT = 7000;
+  private static final ServerRoutingConfig noopConfig = ServerRoutingConfig.create(
```

Comment:
Since this is mutable, should we make it non-static "just in case"?

Reply:
do you mean making `noopConfig` non-static?
Yes i think it makes the verification more strict, which is better.

Reply:
Yes, that's what I meant.